sormas-base/setup/generate-cert.sh

```diff
@@ -69,30 +70,39 @@ echo "The certificate will be generated with the following subject:"
 echo "CN=${SORMAS_S2S_CERT_CN},OU=SORMAS,O=${SORMAS_S2S_CERT_ORG}"
 read -p "Press [Enter] to continue or [Ctrl+C] to cancel."
 
-PEM_NAME=${SORMAS2SORMAS_DIR}/sormas2sormas.privkey.pem
-P12_NAME=${SORMAS2SORMAS_DIR}/sormas2sormas.keystore.p12
-CRT_NAME=${SORMAS2SORMAS_DIR}/sormas2sormas.cert.crt
+PEM_FILE=${SORMAS2SORMAS_DIR}/sormas2sormas.privkey.pem
+P12_FILE=${SORMAS2SORMAS_DIR}/sormas2sormas.keystore.p12
+CRT_FILE=${SORMAS2SORMAS_DIR}/sormas2sormas.cert.crt
 
 # generate private key and self signed certificate
-openssl req -sha256 -newkey rsa:4096 -passout pass:"${SORMAS_S2S_CERT_PASS}" -keyout "${PEM_NAME}" -x509 -passin pass:"${SORMAS_S2S_CERT_PASS}" -days 1095 -subj "${CERT_SUBJ}" -out "${CRT_NAME}"
+openssl req -sha256 -newkey rsa:4096 -passout pass:"${SORMAS_S2S_CERT_PASS}" -keyout "${PEM_FILE}" -x509 -passin pass:"${SORMAS_S2S_CERT_PASS}" -days 1095 -subj "${CERT_SUBJ}" -out "${CRT_FILE}"
```

Comment:
There seems to be a miminum length of 4 characters for the password. Maybe we should check that.